modules/ray_tracing/doc/sqa_reports.yml

```diff
@@ -0,0 +1,39 @@
+Applications:
+    ray_tracing:
+        exe_directory: ${MOOSE_DIR}/modules/ray_tracing
+        app_types:
+            - RayTracingApp
+        content_directory: ${MOOSE_DIR}/modules/ray_tracing/doc/content
+        log_missing_markdown: WARNING
```

Comment:
It would be amazing if this wasn't needed.

Reply:
```
ray_tracing OK
     log_missing_markdown: 0
        log_hidden_syntax: 0
           log_stub_files: 0
      log_duplicate_files: 0
  log_missing_description: 0
```
:-)

Reply:
```
log_missing_markdown: WARNING
```
is removed